.travis.yml

```diff
@@ -10,9 +11,9 @@ before_script:
   - unzip /tmp/radiotools.zip
   - mv radiotools-master radiotools
   - export PYTHONPATH=$PWD/radiotools
-  - wget https://github.com/nu-radio/NuRadioMC/archive/master.zip -O /tmp/NuRadioMC.zip
+  - wget https://github.com/nu-radio/NuRadioMC/archive/python3.zip -O /tmp/NuRadioMC.zip
   - unzip /tmp/NuRadioMC.zip
-  - mv NuRadioMC-master NuRadioMC
```

Comment:
We should rememer to change this right before submission/after NuRadioMC is merged.